sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSource.cs

```diff
@@ -175,16 +175,16 @@ protected EventHubsEventSource() : base(EventSourceName)
         /// <param name="endingSequenceNumber">The sequence number of the last event in the batch.</param>
         /// <param name="durationSeconds">The total duration that the receive operation took to complete, in seconds.</param>
         ///
-        [Event(7, Level = EventLevel.Informational, Message = "Completed receiving events for Event Hub: {0} (Consumer Group: '{1}', Partition Id: '{2}'); Operation Id: '{3}'.  Service Retry Count: {4}; Event Count: {5}; Starting sequence number: '{7}', Ending sequence number: '{8}'; Duration: '{6:0.00}' seconds")]
+        [Event(7, Level = EventLevel.Informational, Message = "Completed receiving events for Event Hub: {0} (Consumer Group: '{1}', Partition Id: '{2}'); Operation Id: '{3}'.  Service Retry Count: {4}; Event Count: {5}; Duration: '{6:0.00}' seconds; Starting sequence number: '{7}', Ending sequence number: '{8}'.")]
```

Comment:
Should the order (duration at the end) still be the same or is it fine to switch it?

Reply:
This should change the display message only while keeping the parameter positions stable in the structured args that ETW emits. There's a small chance that we may impact someone parsing the message text, but we don't generally consider log messages to be part of our official contract nor changes to them breaking.  

In this case, it's the lesser evil; the other option would be to preserve the message display order but change the ordering of parameters.  This is structured data that customers are more likely to be reading directly and there's a higher chance that there is a direct dependency on the index for each argument being referenced.   I'm afraid that changing the order here has a higher potential to break applications.

Reply:
Ah okay I got it; I agree that this is the lesser of two evils for this scenario.